internal/server/recon/golden/find_entities_test.go

```diff
@@ -45,6 +45,12 @@ func TestFindEntities(t *testing.T) {
 				},
 				"santa_clara_county.json",
 			},
+			{
```

Comment:
Also, perhaps we should note that ME also refers to the US State of Maine. So some comment about why we match to country/MNE over the state in the US is important.

Reply:
Hmm.. Upon further investigation, country/MNE has a population < 700k while state of Maine has a population > 1.3mn. So why is country/MNE preferred?

Reply:
Added test which includes both "ME" and "me" in: //internal/server/recon/golden/recognize_places_test.go

Now the tests for RecognizePlaces are scattered in a few places, mainly due to V1->V2 API migrations. We'll try to clean it up, deprecate V1 things, then move all tests under V2.

Reply:
Re country/state orders, copy another comment below:

1. RecognizePlaces sort places by population.
2. However, ResolveDescription (which calls BulkFindEntities in find_entities.go) doesn't sort the resolved entities. If needed, we can also sort (maybe in a future PR) by population, but I'm not sure if that's necessary, because ResolveDescription can also take placeType as an input to filter, so if the user really wants "State" or "Country", he/she can specify it in the query, like "<-description{typeOf:State}->dcid"

Reply:
I would vote for making ResolveDescription result match the same logic as RecognizePlaces, but that's outside the scope of this PR.

Reply:
So basically keep the ranking by population - cool.